### ISSUE TITLE
can i set track subscription rule to selectively allow someone (specific participant sid) to subscribe my audio/video track in group room?

### ISSUE THREAD

**Issue by jdsapariya18** [twilio/twilio-video.js#1588]
hello,
i need help,

i'm building group class using video group room and i wants to provide the user a option to show his audio/video to only the instructor (host), i have participant sid of that participant and No other user can subscribe to his audio/video track.

for example.

we have total 5 user in room and one of them is host ( host, user1, user2, user3, user4)
now user2 wants to only show his audio/video to the host.
and no other user can see user2 audio/video.
can twilio do that by using REST api to set the track subscription rule by setting rule to something like exclude that person and include all for everyone?

or is there any proper way to do that?

**Comment by kumail-raza** [comment]
Nice case. Twilio user here just like you.

Approach 1: You can right your own logic such that when user2 want to go one on one with Host, you can remove the tracks of him from DOM for other users and keep it for Host user.

Approach 2: (Quite complex may be and may be challenging) You can create another room and where Host and the only user will join for a while.

**Comment by jdsapariya18** [comment]
yes, your approach is also in my mind, i also have my node socket connected in that page so i will just put button for user, on the click of that button i will just send socket request to that room and all other participant will listen to the event.

so on getting the socket event every other participant will check that is he participant or host, if he is participant then i will detach or just remove the video & autdio stream from dom.

now the problem is internet bandwidth, i think not using the streams are still using my internet bandwidth, that's why i need proper solution where i tell twilio to only send my tracks (audio/video) to specific person only.
